overlord/snapstate/target.go

```diff
@@ -754,3 +818,467 @@ func validatedComponentInfo(path string, si *snap.Info, csi *snap.ComponentSideI
 	}
 	return componentInfo, nil
 }
+
+// UpdateSummary contains the data that describes an update, including a list of
+// Target structs that represent the snaps that are to be updated.
+type UpdateSummary struct {
```

Comment:
does this needs to be exported?  Summary is really not a great name for this, we use summary mostly for descriptive strings we attach to Tasks and Changes

Reply:
No, it can be un-exported. Any suggestions? `updateManifest`?

Reply:
updateTargets  (we do have some plural structs) or updateInstruction or updatePlan, updateSetup ?